journal-letters/modernized/15-chapter-14.adoc

```diff
@@ -89,8 +92,12 @@ where we were received with joy and gladness by Friends,
 who rejoiced with us in the good service which we had had;
 and blessed be the Lord for our preservation.
 
+[.offset]
 +++[+++From this city William Caton addressed a letter to Friends in England,
-(as is supposed,) from which the following extracts are selected.]
+(as is supposed,) from which the following extracts are selected.+++]+++
+
+[.embedded-content-document.letter]
+--
```

Comment:
Let's make the `O! My beloved friends` line below a `[.saluation]` -- to match what you did in the original, that seems best to me.  It's a little bit fuzzy whether it's a salutation or not, but I lean towards making it one.